lib/registry.js

```diff
@@ -121,7 +121,7 @@ Registry.prototype.createModel = function(name, properties, options) {
   var model = BaseModel.extend(name, properties, options);
   model.registry = this;
 
-  this._defineRemoteMethods(model, options.methods);
+  this._defineRemoteMethods(model, model.settings.methods);
```

Comment:
Interesting. So this assumes that when a new model is created, juggler will propagate `settings.methods` from the parent model to the child model.

What will happen in this case? Could you please add a test for that?

```js
var BaseModel = loopback.createModel('BaseModel', {}, {
  methods: {
    greet: {
      http: { path: '/greet' },
    },
  },
});

var ChildModel = loopback.createModel('ChildModel', {}, {
  methods: {
    hello: {
      http: { path: '/hello' },
    },
  },
});

// ChildModel should have both "greet" and "hello"
```